lib/python/vdm/server/HTTPListener.py

```diff
@@ -480,6 +483,41 @@ def map_deployment_users(request, user):
     return deployment_user[0]
 
 
+def ignore_sighup():
+    signal.signal(signal.SIGHUP, signal.SIG_IGN)
+    signal.signal(signal.SIGINT, signal.SIG_IGN)
```

Comment:
Because Ctrl-C will be for VDM, not Volt server. If we don't ignore it, if user Ctrl-C's out of vdm, volt server will also be stopped.